geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreTest.scala

```diff
@@ -82,9 +82,10 @@ class AccumuloDataStoreTest extends Specification with TestWithMultipleSfts {
       ds.getSchema(defaultSft.getTypeName) mustEqual defaultSft
     }
 
-    "reject a schema with a ~" in {
+    "escape a ~ in the feature name" in {
       val sft = SimpleFeatureTypes.createType("name~name", "name:String,geom:Point:srid=4326")
-      ds.createSchema(sft) must throwAn[IllegalArgumentException]
+      ds.createSchema(sft)
```

Comment:
could you add a `ds.getSchema("name~name") must not(beNull)` and `ds.getSchema("name~name").getTypeName mustEqual "name~name"` ?